README.md

```diff
@@ -60,8 +72,24 @@ Requirements
 
 ci-dnf-stack works on Python 2.7.
 
+For local tests only docker is required and can be installed and set as follows:
+```
+$ sudo dnf install docker
+$ sudo usermod -a -G dockerroot jenkins
+```
+
+Edit /etc/sysconfig/docker where replace line OPTIONS='--selinux-enabled' with OPTIONS='--selinux-enabled -G dockerroot'
```

Comment:
It is required to run docker without sudo. That mean that cidnfstack.py is possible to run without sudo privilege with docker tests.